rotkehlchen/tests/api/test_balances.py

```diff
@@ -717,11 +717,11 @@ def test_balances_caching_mixup(
             task_id=task_id_eth,
             timeout=ASYNC_TASK_WAIT_TIMEOUT * 2,
         )
-        assert result_eth['per_account']['ETH'][ethereum_accounts[0]]['assets']['ETH']['amount'] == '1'  # noqa: E501
-        assert result_eth['per_account']['ETH'][ethereum_accounts[0]]['assets'][A_RDN.identifier]['amount'] == '2'  # noqa: E501
+        assert result_eth['per_account']['eth'][ethereum_accounts[0]]['assets']['ETH']['amount'] == '1'  # noqa: E501
```

Comment:
so like here use this to remove confusion (and same in other places)

```suggestion
        assert result_eth['per_account']['eth'][ethereum_accounts[0]]['assets'][A_ETH.identifer]['amount'] == '1'  # noqa: E501
```